docs/config/README.md

```diff
@@ -72,6 +83,7 @@ minio server /data
 ```
 
 #### Worm
+
```

Comment:
Worm cannot be set through config anymore. This section should be removed.

Reply:
I see worm field in config v31, AFAIK only browser and domain are env var only settings.

Reply:
Yes its possible to set it through config @poornas @nitisht since its not a handler style issue we have for domain or browser.